packages/core/src/renderer.ts

```diff
@@ -747,6 +778,66 @@ export class Renderer {
         };
         break;
       }
+      case "SENDETHTRANSACTION": {
+        const promise = new Promise<void>((resolve) => {
+          setTimeout(() => {
+            this.onSendEthTransactionAction(
+              {
+                data: {
+                  from: this.replaceInlineContext(action.txData.from),
+                  to: this.replaceInlineContext(action.txData.to),
+                  value: this.replaceInlineContext(action.txData.value || "0"),
```

Comment:
we need to be fairly clear that these txData values won't ever inherently have `{{something}}` text in them, otherwise it's going to mess up the transaction in the variable replace

Reply:
Shouldn't be a problem, all the fields are hex data